docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2642/2642_5.md

```diff
@@ -0,0 +1,42 @@
+# Check running instances restriction for Cluster runs
+
+Test verifies that the restriction of the running instances count works correctly for case of Cluster runs
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+
+**Preparations**:
+1. Login as the admin user from the prerequisites
+2. Open the **Settings** page
+3. Open the **USER MANAGEMENT** tab
+4. Click the **Users** tab
+5. Find non-admin user and open it to edit
+6. In the *Launch options* section input value `3` into the *Allowed instance max count* field
+7. Click **OK** button
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the non-admin user from the prerequisites | |
+| 2 | Open the **Tools** page | |
+| 3 | Select the test tool | |
+| 4 | Launch a tool with default settings | |
+| 5 | Repeat steps 2-3 | |
+| 6 | Select the test tool | |
+| 7 | Hover over **v** button near the **Run** button. Select the **Custom settings** item | |
+| 8 | Expand the **Exec environment** section | |
+| 9 | Click the **Configure cluster** control | |
+| 10 | Click the **Cluster** tab | |
+| 11 | Set value `2` into ***Child nodes:*** field | Error message `You have exceeded maximum number of running jobs (3).` is shown on the ***Configure cluster*** pop-up, where `3` is the limit specified at step 6 of the Preparations |
+| 12 | Click the **OK** button | Error message `You have exceeded maximum number of running jobs (3).` is shown in the ***Exec environment*** section |
+| 13 | Click **Launch** button | **Launch** pop-up that contains error message `You have exceeded maximum number of running jobs (3).` appears | 
+| 14 | Click **Launch** button | Error message `Launch of new jobs is restricted as [<non-admin user>] user will exceed [<user-contextual-limit>] runs limit [3]` appears, where <br> `<non-admin user>` is the non-admin user from the prerequisites, <br> `3` is the value specified at step 6 of the Preparations |
+| 15 | Repeat steps 9-10 | |
+| 16 | Set value `1` into ***Child nodes:*** field | Error message disappeared | |
+| 17 | Launch tool | |
+| 18 | Select the test tool | |
+| 19 | Launch a tool with default settings | Error message `Launch of new jobs is restricted as [<non-admin user>] user will exceed [<user-contextual-limit>] runs limit [3]` appears |
+| 20 | Click the run launched at step 4 | |
+| 21 | Wait until the **SSH** hyperlink appears | |
+| 22 | Click the **SSH** hyperlink | |
+| 23 | Enter and perform the command: `pipe users instances` | Response contains follow info: <li> `Active runs detected for a user: [<non-admin user>: 3]` <li> `The following restriction applied on runs launching: [<non-admin user>: 3]` |
```

Comment:
I've tried to perform a test case, and the response I've got - `Active runs detected for a user: [<non-admin user>: 2] ...`

Reply:
This's result the bug described in https://github.com/epam/cloud-pipeline/issues/2642#issuecomment-1171475127